apps/web/pages/settings/my-account/calendars.tsx

```diff
@@ -1,11 +1,12 @@
 import { Trans } from "next-i18next";
 import Link from "next/link";
 import { useRouter } from "next/navigation";
-import { Fragment } from "react";
+import React, { Fragment, useState, useEffect } from "react";
```

Comment:
```suggestion
import { Fragment, useState, useEffect } from "react";
```

I dont think we need to `import React` here